src/components/pages/alerts/AlertsComponent.jsx

```diff
@@ -1,12 +1,76 @@
 import Box from '@material-ui/core/Box';
-
-import SearchBar from './SearchBar';
 import Alert from './Alert';
 
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import DateFnsUtils from '@date-io/date-fns';
+import {
+  MuiPickersUtilsProvider,
+  KeyboardTimePicker,
+  KeyboardDatePicker,
+} from '@material-ui/pickers';
+
 const AlertsComponent = () => {
+   const [selectedDateFrom, setSelectedDateFrom] = React.useState(new Date());
+   const [selectedDateTo, setSelectedDateTo] = React.useState(new Date());
+
+   const handleDateChangeFrom = (date) => {
+     setSelectedDateFrom(date);
+   };
+
+   const handleDateChangeTo = (date) => {
+     setSelectedDateTo(date);
+   };
+
    return (
       <Box padding="3vh 10vh 3vh 10vh">
-         <SearchBar />
+         <MuiPickersUtilsProvider utils={DateFnsUtils}>
+            <Grid container justify="space-around">
```

Comment:
Grids are mainly used for when you're structuring horizontally _and_ vertically. You could use a Box instead.

Reply:
Fixed